lightly_studio/src/lightly_studio/api/routes/api/sample.py

```diff
@@ -47,6 +47,8 @@ def read_samples(
     Returns:
         A list of filtered samples.
     """
+    if body.filters.dataset_id is None:
```

Comment:
Minor: Is `dataset_id` optional? If so, why is it defined as optional but required in the end?